### ISSUE TITLE
Consider prefixing @private methods with `_`

### ISSUE THREAD

**Issue by cflynn07** [Runnable/scientist#4]
Really helps organization & module readability

**Comment by cflynn07** [comment]
Ex: https://github.com/Runnable/scientist/blob/master/lib%2Fexperiment.js#L60

**Comment by albertorestifo** [comment]
I disagree for the same reason [they do](https://github.com/airbnb/javascript#naming--leading-underscore)
